docs/customresourcestate-metrics.md

```diff
@@ -480,4 +480,7 @@ Examples:
 
 # if the value to be matched is a number or boolean, the value is compared as a number or boolean  
 [status, conditions, "[value=66]", name]  # status.conditions[1].name = "b"
+
+# expand a list
+[spec, order, "*", value] # spec.order[*].value = true
```

Comment:
Do you have an example of use-case for that?

As far as I can tell, the best practice would be to have one metric per value in the list no?

Reply:
With `*`, the list is made available for further filtering, so for the YAML snippet below, 

```yaml
status:
  namespaces:
    - namespace: "foo"
      status:
        available:
          resourceA: '10'
          resourceB: '20'
        pending:
          resourceA: '0'
          resourceB: '6'
```
and the configuration below,
```yaml
path: [status, namespaces, "*", status]
labelsFromPath:
  "available_*": [available]
```
would yield the metric,
```yaml
...{...,available_resourceA="10",available_resourceB="20"} ...
```

Reply:
With a non-wildcard string, and the configuration below,
```yaml
path: [status, namespaces, "*", status]
labelsFromPath:
  "label": [available]
```
would yield the metric,
```yaml
...{...,label0="10",label1="20"} ...
```